src/actions.hpp

```diff
@@ -45,20 +45,13 @@ namespace Action {
 // *****************************************************************************
 // class definitions
 
-    /*!
-      @brief Abstract base class for all concrete actions.
-
-      Task provides a simple interface that actions must implement and a few
-      commonly used helpers.
-     */
+    //! @brief Interface for all concrete actions.
     class Task {
     public:
         //! Shortcut for an auto pointer.
-        typedef std::unique_ptr<Task> UniquePtr;
```

Comment:
Any particular reason why you removed this one?

Reply:
The only reason is that I do not like typedefs that are not used a lot. In this case I could remove 7-8 lines from the header file by just removing these typedefs. 

It also seems to me a bit unnatural to have 7-8 typedefs called `UniquePtr` in the same file, which are unique pointers to different types.

Reply:
Makes sense, since this is not a public class, let's drop it then.